source/extensions/filters/network/redis_proxy/proxy_filter.cc

```diff
@@ -28,6 +28,12 @@ ProxyFilterConfig::ProxyFilterConfig(
           Config::DataSource::read(config.downstream_auth_username(), true, api)),
       dns_cache_manager_(cache_manager_factory.get()), dns_cache_(getCache(config)) {
 
+  if (config.settings().enable_redirection() && !config.settings().has_dns_cache_config()) {
+    ENVOY_LOG(
+        warn,
+        "redirections enabled without DNS lookups so some responses might generate client errors");
```

Comment:
Sure, is something like:

```
redirections without DNS lookups enabled might cause client errors, set the dns_cache_config field within the connection pool settings to avoid them
```

good enough?